contracts/protocol/tokenization/AToken.sol

```diff
@@ -59,7 +59,7 @@ contract AToken is VersionedInitializable, ScaledBalanceTokenBase, EIP712Base, I
     string calldata aTokenName,
     string calldata aTokenSymbol,
     bytes calldata params
-  ) external override initializer {
+  ) public virtual override initializer {
```

Comment:
This needs to be public as well as virtual for extensions to be able to call `super.initiailize(...)`